1/contrib/openshift/configuration/jobs/OpenShift Sample/config.xml

```diff
@@ -13,66 +13,24 @@
   <builders>
 
     <com.openshift.jenkins.plugins.pipeline.OpenShiftScaler>
-      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
-      <depCfg>frontend</depCfg>
-      <namespace>test</namespace>
-      <replicaCount>0</replicaCount>
-      <authToken></authToken>
     </com.openshift.jenkins.plugins.pipeline.OpenShiftScaler>
 
-    <com.openshift.jenkins.plugins.pipeline.OpenShiftDeploymentVerifier>
-      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
-      <depCfg>frontend</depCfg>
-      <namespace>test</namespace>
-      <replicaCount>0</replicaCount>
-      <authToken></authToken>
-    </com.openshift.jenkins.plugins.pipeline.OpenShiftDeploymentVerifier>
-
     <com.openshift.jenkins.plugins.pipeline.OpenShiftBuilder>
-      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
-      <bldCfg>frontend</bldCfg>
-      <namespace>test</namespace>
-      <authToken></authToken>
-      <followLog>true</followLog>
     </com.openshift.jenkins.plugins.pipeline.OpenShiftBuilder>
 
-    <com.openshift.jenkins.plugins.pipeline.OpenShiftScaler>
-      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
-      <depCfg>frontend</depCfg>
-      <namespace>test</namespace>
-      <replicaCount>1</replicaCount>
-      <authToken></authToken>
-    </com.openshift.jenkins.plugins.pipeline.OpenShiftScaler>
```

Comment:
I can docker run new image and confirm setting on steps

On Friday, February 12, 2016, Gabe Montero <gmontero@redhat.com> wrote:

> Yep i went too far in pruning - given how the image is pulled during
> deploy, i constructed the new job manually.  I'll push correction shortly.
>
> On Friday, February 12, 2016, Ben Parees <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
>> In 1/contrib/openshift/configuration/jobs/OpenShift Sample/config.xml
>> <https://github.com/openshift/jenkins/pull/79#discussion_r52779646>:
>>
>> >      </com.openshift.jenkins.plugins.pipeline.OpenShiftBuilder>
>> >
>> > -    <com.openshift.jenkins.plugins.pipeline.OpenShiftScaler>
>> > -      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
>> > -      <depCfg>frontend</depCfg>
>> > -      <namespace>test</namespace>
>> > -      <replicaCount>1</replicaCount>
>> > -      <authToken></authToken>
>> > -    </com.openshift.jenkins.plugins.pipeline.OpenShiftScaler>
>>
>> we should still scale to 1, since we scaled to zero earlier we need to
>> scale it back up (technically we might not need to since DC logic might set
>> it back to 1, but it seems saner to make it explicit)
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/openshift/jenkins/pull/79/files#r52779646>.
>>
>